cmake/Modules/FindBLIS.cmake

```
@@ -0,0 +1,62 @@
+SET(BLIS_INCLUDE_SEARCH_PATHS
+ /usr/include
+ /usr/include/blis
+ /usr/local/include
+ /usr/local/include/blis
+ /usr/local/opt/blis/include
+ /opt/blis/include
+ $ENV{BLIS_HOME}
+ $ENV{BLIS_HOME}/include
+)
+
+SET(BLIS_LIB_SEARCH_PATHS
+ /lib
+ /lib/blis
+ /lib64
+ /lib64/blis
+ /usr/lib
+ /usr/lib/blis
+ /usr/lib64
+ /usr/lib64/blis
+ /usr/local/lib
+ /usr/local/lib64
+ /usr/local/opt/blis/lib
+ /opt/blis/lib
+ $ENV{BLIS}
+ $ENV{BLIS}/lib
+ $ENV{BLIS_HOME}
+ $ENV{BLIS_HOME}/lib
```

Please do not include standard library folders in extra search path
Also, can you explain why `blis` might get installed in `/lib/ or `/lib64` folder (which is reserved for core OS libraries, isn't it?)
```suggestion
 /lib/blis
 /lib64/blis
 /usr/lib/blis
 /usr/lib64/blis
 /usr/local/opt/blis/lib
 /opt/blis/lib
 $ENV{BLIS}
 $ENV{BLIS}/lib
 $ENV{BLIS_HOME}
 $ENV{BLIS_HOME}/lib
```

---

Addressed the comments

---

We had followed the OpenBLAS convention for library files :
SET(Open_BLAS_LIB_SEARCH_PATHS
        /lib/
        /lib/openblas-base
        /lib64/
        /usr/lib
        /usr/lib/openblas-base
        /usr/lib64
        /usr/local/lib
        /usr/local/lib64
        /usr/local/opt/openblas/lib
        /opt/OpenBLAS/lib
        $ENV{OpenBLAS}
        $ENV{OpenBLAS}/lib
        $ENV{OpenBLAS_HOME}
        $ENV{OpenBLAS_HOME}/lib
 )